pkgs/development/interpreters/perl/default.nix

```diff
@@ -77,6 +77,7 @@ let
         "-Dlocincpth=${libcInc}/include"
         "-Dloclibpth=${libcLib}/lib"
       ]
+      ++ optionals ((builtins.match "5\.[0-9]*[13579]\..+" version) != null) [ "-Dusedevel" "-Uversiononly" ]
```

Comment:
While this regex works for now, it doesn't seem fully correct (I assume you want to distinguish odd/even minor versions):

```
nix-repl> builtins.match "5\.[0-9]*[13579]\..+" "5.26.123"
null

nix-repl> builtins.match "5\.[0-9]*[13579]\..+" "5.27.123"
[ ]

nix-repl> builtins.match "5\.[0-9]*[13579]\..+" "5.28.123"
null

nix-repl> builtins.match "5\.[0-9]*[13579]\..+" "5.29.123"
[ ]

nix-repl> builtins.match "5\.[0-9]*[13579]\..+" "5.30.123"
[ ]

nix-repl> builtins.match "5\.[0-9]*[13579]\..+" "5.31.123"
[ ]
```

Reply:
```
nix-repl> builtins.match "5\.[0-9]*[13579]\..+" "5.30.123"
[ ]
```
yes, this expected to be ```null```

Reply:
```
nix-repl> builtins.match "5\.[0-9]*[13579]\..+" "5.30.123"
[ ]
```
yes, this expected to be ```null```